src/pencilkit.cs

```diff
@@ -246,9 +300,171 @@ interface PKToolPicker {
 		[Export ("colorUserInterfaceStyle", ArgumentSemantic.Assign)]
 		UIUserInterfaceStyle ColorUserInterfaceStyle { get; set; }
 
+		[Introduced (PlatformName.iOS, 13, 0, message: "Create individual instances instead.")]
+		[Deprecated (PlatformName.iOS, 14, 0, message: "Create individual instances instead.")]
 		[Static]
 		[return: NullAllowed]
 		[Export ("sharedToolPickerForWindow:")]
 		PKToolPicker GetSharedToolPicker (UIWindow window);
+
+		[iOS (14, 0)]
+		[Introduced (PlatformName.MacCatalyst, 14, 0)]
+		[Export ("showsDrawingPolicyControls")]
+		bool ShowsDrawingPolicyControls { get; set; }
+
+		// !missing-null-allowed! 'System.Void PencilKit.PKToolPicker::set_AutosaveName(System.String)' is missing an [NullAllowed] on parameter #0
+		[iOS (14, 0)]
+		[Introduced (PlatformName.MacCatalyst, 14, 0)]
+		[Export ("stateAutosaveName")]
+		string stateAutosaveName { get; set; }
+	}
+
+	[Mac (10, 16), iOS (14, 0)]
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
+	[BaseType (typeof(NSObject))]
+	interface PKInk : NSCopying
+	{
+		[Export ("initWithInkType:color:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string type, UIColor color);
+
+		[Export ("inkType")]
+		string InkType { get; }
+
+		[Export ("color")]
+		UIColor Color { get; }
+	}
+
+	[Mac (10, 16), iOS (14, 0)]
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
+	[BaseType (typeof(NSObject))]
+	interface PKFloatRange : NSCopying
+	{
+		[Export ("initWithLowerBound:upperBound:")]
+		IntPtr Constructor (nfloat lowerBound, nfloat upperBound);
+
+		[Export ("lowerBound")]
+		nfloat LowerBound { get; }
+
+		[Export ("upperBound")]
+		nfloat UpperBound { get; }
+	}
+
+	[Mac (10, 16), iOS (14, 0)]
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
+	[BaseType (typeof(NSObject))]
+	interface PKStroke : NSCopying
+	{
+		[NoMac]
+		[Export ("initWithInk:strokePath:transform:mask:")]
+		IntPtr Constructor (PKInk ink, PKStrokePath path, CGAffineTransform transform, [NullAllowed] UIBezierPath mask);
+
+		[NoiOS]
+		[Export ("initWithInk:strokePath:transform:mask:")]
+		IntPtr Constructor (PKInk ink, PKStrokePath path, CGAffineTransform transform, [NullAllowed] NSBezierPath mask);
+
+		[Export ("ink")]
+		PKInk Ink { get; }
+
+		[Export ("transform")]
+		CGAffineTransform Transform { get; }
+
+		[Export ("path")]
+		PKStrokePath Path { get; }
+
+#if MONOMAC
+		[NoiOS]
+		[NullAllowed, Export ("mask")]
+		NSBezierPath Mask { get; }
+#else
+		[NoMac]
+		[NullAllowed, Export ("mask")]
+		UIBezierPath Mask { get; }
+#endif
+
+		[Export ("renderBounds")]
+		CGRect RenderBounds { get; }
+
+		[Export ("maskedPathRanges")]
+		PKFloatRange[] MaskedPath { get; }
```

Comment:
I think we should keep the `Ranges` part of the name, it's not a suffix (and the swift version of the API also has it):

```suggestion
		PKFloatRange[] MaskedPathRanges { get; }
```

Reply:
Agreed, as a general rule always keep property names just as the ObjC ones and when in doubt look at the swift name.